### PR TITLE
fix groupId misalignment

### PR DESCRIPTION
**Pull request by devenyantis**
When you added a new program / programs , put a cross on the  OS that you have test it on  
(Example: [ X ] Windows  )

[  ] Windows  
[ X ] Linux Ubuntu  
[  ] Mac  
[  ] Others  (State here -> xxx )  

Assign a reviewer to review your code.

### CODE REVIEW

**Comment by devenyantis** [comment]
I think this also solve for #76

File: my-first-dl4j-project/pom.xml [hunk 1]
```diff
@@ -26,8 +26,8 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>my-first-dl4j-project</groupId>
-  <artifactId>global.skymind</artifactId>
+  <groupId>ai.certifai</groupId>
+  <artifactId>my-first-dl4j-project</artifactId>
   <version>1.0-SNAPSHOT</version>
 
   <name>global.skymind</name>
```
Comment (codenamewei) [inline my-first-dl4j-project/pom.xml:33]: This should be same with artifactId @devenyantis

**Comment by codenamewei** [comment]
@devenyantis 
Import my-first-dl4j-project as standalone and run as in the readme.
https://github.com/CertifaiAI/TrainingLabs/blob/groupId_misalignment/my-first-dl4j-project/readme.md

You would realize that the jar file name changed as well.

Change it accordingly in the following
- https://github.com/CertifaiAI/TrainingLabs/blob/groupId_misalignment/my-first-dl4j-project/pom.xml#L113
- https://github.com/CertifaiAI/TrainingLabs/blob/groupId_misalignment/my-first-dl4j-project/readme.md

Also elaborate more on the readme...
For example 
- what it means as a standalone project
- how to use this particular folder (people do not realize they can just copy and paste this out to be the skeleton base of their project, it does not related to any other modules of TrainingLabs) 
- explaining what is an uberjar
- how to run (it can be run as it is by running it on ide or build as uberjar run with java commands in the terminal)

**Comment by devenyantis** [comment]
- change name tag
- add more description to readme

Tested on Ubuntu 18.04

**Pull request merged**